postgresql/shell-scripts/unix/create-db-changelog.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+# Shell script to create a changelog extension of the 3DCityDB
+# on PostgreSQL/PostGIS
+
+# Get the current directory path of this script file
+CURRENT_DIR="$( cd "$( dirname "$0" )" && pwd )"
+if [ $# -ne 0 ]; then
+  source "$1"
+else
+  if [ -f connection-details.sh ]; then
+	  source connection-details.sh
+  else
+	  source "$CURRENT_DIR/connection-details.sh"
+  fi
+fi
+
+# Add PGBIN to PATH
+export PATH="$PGBIN:$PATH"
+
+# Welcome message
+echo ' _______   ___ _ _        ___  ___ '
+echo '|__ /   \ / __(_) |_ _  _|   \| _ )'
+echo ' |_ \ |) | (__| |  _| || | |) | _ \'
+echo '|___/___/ \___|_|\__|\_, |___/|___/'
+echo '                     |__/          '
+echo
+echo '3D City Database - The Open Source CityGML Database'
+echo
+echo '######################################################################################'
+echo
+echo 'Welcome to the 3DCityDB Setup Script. This script will guide you through the process'
+echo 'of setting up a 3DCityDB instance. Please follow the instructions of the script.'
```

Comment:
Please adapt the description. This is obviously a copy of the create-db script. But this script does not setup a 3DCityDB instance

Reply:
Fixed with [f85d574](https://github.com/3dcitydb/3dcitydb/pull/226/commits/f85d57478a4343404e944d8b1d2a5231ff47bfe4)